docs/translation.md

```diff
@@ -43,15 +48,21 @@ sequenceDiagram
   Developer ->> GitHubActions: Merge pull request into develop
   GitHubActions ->> GitHubActions: Scan codebase and write strings to .po file
   GitHubActions ->> GitHubActions: Fill .po file with existing translations
-  GitHubActions ->> GitHub: Commit .po file with updated strings
-  GitHubActions ->> Poeditor: Call webhook
-  Poeditor ->> GitHub: Fetch updated .po file
-  Poeditor ->> Poeditor: Synchronize translations with latest strings and translations
-  Translator ->> Poeditor: Translate strings
-  Translator ->> Poeditor: Export strings to GitHub
-  Poeditor ->> GitHub: Commit .po file with updated translations to develop
+  Note over GitHubActions,GitHubActions: if config translations<br/>are available<br/>they are used<br/>otherwise empty strings
+  GitHubActions ->> GitHubActions: Exports configuration translations into a .config.po file
+  GitHubActions ->> GitHubActions: The two .po files are merged together into a .combined.po file
+  GitHubActions ->> GitHub: Commit combined.po file with updated strings
+  GitHubActions ->> POEditor: Call webhook
+  POEditor ->> GitHub: Fetch updated combined.po file
+  POEditor ->> POEditor: Synchronize translations with latest strings and translations
+  Translator ->> POEditor: Translate strings
+  Translator ->> POEditor: Export strings to GitHub
+  POEditor ->> GitHub: Commit combined.po file with updated translations to develop
+  Github ->> Github: .combined.po is split into two files: .po and .config.po
+  Github ->> Github: All the po files are published to Github Pages
```

Comment:
```suggestion
  GitHub ->> GitHub: .combined.po is split into two files: .po and .config.po
  GitHub ->> GitHub: All the po files are published to Github Pages
```

Note that participant names are case sensitive. You are creating a new participant here "Github" :smile: